### PR TITLE
User model

### PR DESCRIPTION
**Pull request by anaturrillo**
## Summary

I created a Sequelize user model and corresponding migration. 
Running the command ```npx sequelize-cli model:generate --name ... ```  line did not allow any configuration other than data type ([here's a thread about it](https://github.com/sequelize/cli/issues/215)). What I did was to execute the command to have a base to work with and then I altered it manually. I'm not sure if that was the intended procedure. Though I still have some doubts regarding Sequelize I was able to run the migrations script and a users table was created :)

## Trello Card

[https://trello.com/c/LHxshZrJ/26-modelo-de-usuario] (https://trello.com/c/LHxshZrJ/26-modelo-de-usuario)

### CODE REVIEW

File: app/models/index.js [hunk 1]
```diff
@@ -21,7 +21,9 @@ Object.keys(db).forEach(modelName => {
   }
 });
 
+// esta db
 db.sequelize = sequelize;
+// modulo Sequelize
```
Comment (mpicollo-wolox) [inline app/models/index.js]: Remove this comments please 😛

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {}
```
Comment (mpicollo-wolox) [inline app/models/user.js]: We use snake_case in database (see [underscored](https://sequelize.org/v4/manual/tutorial/models-definition.html#configuration) option)

File: app/models/user.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  sequelize.define(
+    'user',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true,
+        primaryKey: true
+      },
```
Comment (mpicollo-wolox) [inline app/models/user.js]: You don't need define this (in model definition), sequelize autogenerate id like this

Comment (anaturrillo) [reply app/models/user.js]: I tried not to define it and it takes the email as key:
Screenshot from 2019-08-13 18-18-57

Comment (anaturrillo) [reply app/models/user.js]: I tried to not define id and it takes email as key:
Screenshot from 2019-08-13 18-18-57
Is that what you meant?

File: app/models/user.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      firstName: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        field: 'first-name'
+      },
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        field: 'last-name'
```
Comment (mpicollo-wolox) [inline app/models/user.js]: This is spinal-case, are you sure this work ?

File: app/models/user.js (hunk 2 above)
Comment (mpicollo-wolox) [reply app/models/user.js]: Check this comment

File: migrations/migrations/20190813213033-create-user.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
```
Comment (mpicollo-wolox) [inline migrations/migrations/20190813213033-create-user.js]: This should be un snake_case too

File: migrations/migrations/20190812204643-create-user.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
+'use strict';
```
Comment (mpicollo-wolox) [inline migrations/migrations/20190812204643-create-user.js]: This migration is duplicated

File: app/models/user.js (hunk 4 above)
Comment (anaturrillo) [reply app/models/user.js]: No! I actually corrected it on the other branch were I actually tested the user creation.

**Pull request merged**